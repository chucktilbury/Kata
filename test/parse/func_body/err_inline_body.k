;-------------------------------------------------------------------------------
; An inline expects to have raw text enclosed by {}
;-------------------------------------------------------------------------------

func name()() {
	inline dict stuff
}
