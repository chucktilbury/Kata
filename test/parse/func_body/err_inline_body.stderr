Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_inline_body.k: 6: 9: inline: expected a '{' to introduce the block
Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_inline_body.k: 8: 1: expected function body element but got end of file
Syntax: /home/chuck/Src/Kata/test/parse/func_body/err_inline_body.k: 8: 1: expected function body but got end of file
